src/main/resources/migrations.xml

```diff
@@ -2633,4 +2633,35 @@
 		</addColumn>
 	</changeSet>
 
+	<changeSet id="add-endedAt-to-peoplePositions" author="mdragan">
+		<addColumn tableName="peoplePositions">
+			<column name="endedAt" type="datetime" />
+		</addColumn>
+		<sql dbms="mssql">
+			ALTER TABLE peoplePositions ALTER COLUMN endedAt datetime2;
```

Comment:
Why alter this to `datetime2`?

Reply:
Because that's how it's done in the rest of the migrations too :-)
And it was done like that because: "<!-- Liquidbase uses datetime, but SQLServer recommends using datetime2 instead -->"